documentation/src/main/asciidoc/reference/introduction.adoc

```diff
@@ -497,7 +497,7 @@ Whatever you decide, the first step to getting a reactive session is to obtain
 a JPA `EntityManagerFactory` just as you usually would in plain ol' regular JPA,
 for example, by calling:
 
-[java]
+[source, JAVA, indent=0]
```

Comment:
@DavideD but this doesn't work, does it?

When I tried it I got errors.

Reply:
I don't see any error

Reply:
Ah, well, actually the error is in IntelliJ, so perhaps it's a bug in IntelliJ not in ASCII doctor itself?

Reply:
Indeed, seems to be a bug in the IntelliJ plugin. I had taken it to be a limitation of asciidoc itself.

Reply:
IntelliJ now is giving me two warnings:
* one for `[source, JAVA, indent=0]`
* one for the java code not starting with class or something

I will replace it with `[source,java]` but the other problem is an Intellij thing. Asciidoc doesn't seem to care about it.

Reply:
Ah no wait, forget what I said. IntelliJ was refreshing or something.

It seems an issue with IntelliJ though. I usually use VIM for document changes so I wouldn't know.

Reply:
> It seems an issue with IntelliJ though.

Yes, I agree. So go ahead and merge.